plugins/micron/micron-nvme.c

```diff
@@ -3169,7 +3079,7 @@ static int micron_internal_logs(int argc, char **argv, struct command *cmd,
 		sn[j++] = ctrl.sn[i];
 	}
 	sn[j] = '\0';
-	strcpy(ctrl.sn, sn);
+	strscpy(ctrl.sn, sn);
```

Comment:
Missing `count`

Reply:
This is an un-intended change, reverted back. Thanks for catching!